etc/uams/uams_dhx_pam.c

```diff
@@ -25,19 +25,19 @@
 #endif
 #include <arpa/inet.h>
 
-#if defined(GNUTLS_DHX)
-#include <gnutls/openssl.h>
+#if defined(WOLFSSL_DHX)
+#include <wolfssl/options.h>
+#include <wolfssl/openssl/bn.h>
+#include <wolfssl/openssl/dh.h>
+#include <wolfssl/openssl/err.h>
+#include <wolfssl/openssl/ssl.h>
+#include <atalk/cast.h>
 #elif defined(OPENSSL_DHX)
 #include <openssl/bn.h>
 #include <openssl/dh.h>
 #include <openssl/cast.h>
 #include <openssl/err.h>
 #include "openssl_compat.h"
-#else /* OPENSSL_DHX */
-#include <bn.h>
-#include <dh.h>
-#include <cast.h>
-#include <err.h>
 #endif /* OPENSSL_DHX */
```

Comment:
Nitpick: Perhaps label this as the endif for WOLFSSL_DHX

Reply:
Done